packages/scandipwa/src/route/CategoryPage/CategoryPage.container.js

```diff
@@ -557,6 +579,20 @@ export class CategoryPageContainer extends PureComponent {
         });
     }
 
+    updatePlpType() {
+        const { plpType } = this.props;
+
+        if (plpType.match('-')) {
+            const plpTypes = plpType.split('-');
+
+            this.setState({ defaultPlpType: plpTypes[0] });
```

Comment:
You can set multiple state values in a single call of `setState()`.

Reply:
Fixed

Reply:
Nice, thanks!